Pod/Classes/Highlightr.swift

```diff
@@ -165,7 +167,20 @@ open class Highlightr
     }
     
     //Private & Internal
-    fileprivate func processHTMLString(_ string: String) -> NSAttributedString?
+    /**
+     Execute block safely on main thread, execute block directly if current thread is main thread, otherwise execute block on main queue synchronously.
+     */
+    private func safeMainSync(_ block: @escaping ()->()) {
```

Comment:
This function is still using K&R indentation.

Reply:
:beers::beers:, I updated.